volume/nfs/nfs.go

```diff
@@ -71,26 +79,70 @@ func (v *Volume) DriverName() string {
 }
 
 func (v *Volume) Path() string {
-	return ""
+	v.m.Lock()
+	defer v.m.Unlock()
+	return v.hostFolder
 }
 
 func (v *Volume) Mount() (string, error) {
-	// The return value from this method will be passed to the container
-	return v.name, nil
-}
+	v.m.Lock()
+	defer v.m.Unlock()
 
-func (v *Volume) Unmount() error {
-	return nil
+	v.usedCount++
+	if v.usedCount > 1 {
+		// Already mounted
+		return v.hostFolder, nil
+	}
+	name, err := ioutil.TempDir(NFS_MOUNTS_FOLDER, "")
+	if err != nil {
+		return "", err
+	}
+	v.hostFolder = name
+	// retry=0,timeo=30: Fail if NFS server can't be reached in three second (no retries) - aggressive, but necessary because the Docker daemon becomes unresponsive if the mount command hangs.
+	// nolock:           Don't use NFS locking, because the host's rpc.statd can't be reached at this point since we're already inside the network namespace.
```

Comment:
We are no longer inside the network namespace, so try removing the `nolock` option (and the comment) and see if it still works.